modules/channel-web/src/backend/socket.ts

```diff
@@ -45,7 +45,7 @@ export default async (bp: typeof sdk, db: Database) => {
       const payload = bp.RealTimePayload.forVisitor(userId, 'webchat.typing', { timeInMs: typing, conversationId })
       // Don't store "typing" in DB
       bp.realtime.sendPayload(payload)
-      await Promise.delay(typing)
+      // await Promise.delay(typing)
```

Comment:
By removing this, the typing effect is no longer possible. And we'll definitely need it here when we use a single payload to send message and typing

Reply:
It's missing in this PR, but I wanted to put the waiting in the front-end instead